docs/network-flow-visibility.md

```diff
@@ -486,13 +486,32 @@ about flow record processing. Refer to the
 ## Quick deployment
 
 If you would like to quickly try Network Flow Visibility feature, you can deploy
-Antrea, the Flow Aggregator Service and the ELK Flow Collector on the
+Antrea, the Flow Aggregator Service, the ELK Flow Collector or Grafana Flow Collector on the
 [Vagrant setup](../test/e2e/README.md). You can use the following command:
 
+### Image-building steps
+
+Build required image under antrea by using make command:
+```shell
```

Comment:
fenced code block need to be surrounded by blank lines
same for below

Reply:
fixed